pkg/topology/depthmonitor/depthmonitor_test.go

```diff
@@ -87,17 +87,23 @@ func TestDepthMonitorService(t *testing.T) {
 	})
 
 	t.Run("depth decrease due to under utilization", func(t *testing.T) {
-		topo := &mockTopology{peers: 1}
-		// >50% utilized reserve
-		reserve := &mockReserveReporter{size: 26000, capacity: 50000}
 
-		bs := mockbatchstore.New(mockbatchstore.WithReserveState(&postage.ReserveState{Radius: 3}))
+		defer func(w uint8) {
+			*depthmonitor.MinimumRadius = w
+		}(*depthmonitor.MinimumRadius)
+		*depthmonitor.MinimumRadius = 0
 
 		defer func(w time.Duration) {
 			*depthmonitor.ManageWait = w
 		}(*depthmonitor.ManageWait)
 		*depthmonitor.ManageWait = 200 * time.Millisecond
 
+		topo := &mockTopology{peers: 1}
+		// >50% utilized reserve
+		reserve := &mockReserveReporter{size: 26000, capacity: 50000}
+
+		bs := mockbatchstore.New(mockbatchstore.WithReserveState(&postage.ReserveState{Radius: 3}))
+
```

Comment:
So in this test, I think you can remove L122 to L126. This was added as we were storing the depth on close. But we no longer need that. The current check doesnt make sense actually.